obj.go

```diff
@@ -86,13 +155,13 @@ func (cc *Conn) DeleteObject(o Obj) {
 // GetObj is a legacy method that return all Obj that belongs
 // to the same table as the given one
```

Comment:
Can you add a brief comment to explain the different behavior regarding legacy/non-legacy Obj types please?